jenkins/validate-artifacts/validate-artifacts.jenkinsfile

```diff
@@ -131,6 +131,10 @@ pipeline {
                         PROJECTS = "opensearch opensearch-dashboards"
                     }
 
+                    if (DOCKER_SOURCE == 'Both') {
+                        DOCKER_SOURCE = "dockerhub ecr"
+                    }
```

Comment:
If this is the case I would rather we give an option of :
```
            choices: ["dockerhub", "ecr", "dockerhub ecr"]
```

Reply:
Same as above ^^

Reply:
Got it.